src/Resources/views/storefront/layout/meta.html.twig

```diff
@@ -258,6 +247,19 @@
                     });
                   }
                 }
+                if ('phone' === markerValue) {
+                    var countryCodeSelector = MarkerElement.getAttribute('data-country-code-selector');
+                    var phoneSelector = MarkerElement.getAttribute('data-phone-selector');
+                    window.EnderecoIntegrator.initPhoneServices(
+                        {
+                            phone: `[data-endereco-${markerValue}-form-id="${formId}"] [name="${phoneSelector}"]`,
+                            countryCode: `[data-endereco-${markerValue}-form-id="${formId}"] [name="${countryCodeSelector}"]`
+                        }, {
+                            name: `phs_${prefix}_${formId}`,
+                            addressType: 'general_address_phs'
```

Comment:
This can be removed. The property "addressType" is only relevant to initAMS call. It essentially defines whether we are dealing with a billing and shipping or a general address, the difference being different texts that are displayed in the modal. 
For the phone there is no "addressType".

Instead there is an optional "numberType". Please add it here like in https://github.com/Endereco/js-sdk/blob/master/example/example.html#L613
This setting should not be defined by default, as by default we would use the type from the plugin settings.
But for a custom integration somebody might want to specifically specify the type ("mobile" or "fixedLine") of the phone number field.

Reply:
Check my last commit, I have added numberType to initPhoneServices - it can be changed on demand from phone services data marker level by adding attribute like `data-number-type="fixedLine"`. By default this value is fetched from plugin config if it's not defined in data marker.